test/unit/report_test.rb

```diff
@@ -113,6 +113,23 @@ def setup
       report = TestReport.new
       assert_equal({}, report.metrics)
     end
+
+    test 'can view reports as non-admin user' do
+      #Testing Issue #13920
+      report = ConfigReport.import read_json_fixture('report-empty.json')
```

Comment:
Can you just use `FactoryGirl.create(:config_report)`? Reading from a file will make this test considerably slower